packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -292,6 +319,83 @@ const Autocomplete = <
       testId,
     ]
   );
+
+  const renderVirtualizedRow = ({
+    data,
+    index,
+    style,
+  }: ListChildComponentProps) => {
+    const baseOption = data[index];
+    /**
+     * react-window calculates the absolute positions of the list items, via an inline style, so
+     * we need to add it to each list item that is being rendered in the viewable list window.
+     * See here if you need to know more: https://github.com/bvaughn/react-window?tab=readme-ov-file#why-is-my-list-blank-when-i-scroll
+     */
+    const optionItem = { ...baseOption, props: { ...baseOption.props, style } };
+    return optionItem;
+  };
+
+  const OuterListboxContext = createContext({});
+
+  const OuterListboxElementType = React.forwardRef<HTMLDivElement>(
+    (props, ref) => {
+      const outerProps = React.useContext(OuterListboxContext);
+      return <div ref={ref} {...props} {...outerProps} />;
+    }
+  );
+
+  function useResetCache(length: number) {
+    const ref = React.useRef<VariableSizeList>(null);
+    useEffect(() => {
+      if (ref.current != null) {
+        ref.current.resetAfterIndex(0, true);
+      }
+    }, [length]);
+    return ref;
+  }
+
+  const ListboxComponent = forwardRef<
+    HTMLDivElement,
+    React.HTMLAttributes<HTMLElement>
+  >(function (props, ref) {
+    const { children, ...other } = props;
+    const itemData: React.ReactElement[] = [];
+    (children as React.ReactElement[]).forEach(
+      (item: React.ReactElement & { children?: React.ReactElement[] }) => {
+        itemData.push(item);
+        itemData.push(...(item.children || []));
+      }
+    );
+
+    const optionHeight = 45; //px
```

Comment:
Where's this `45` coming from? Is it possible to locate it somewhere else in the file so it's easier for the design folks to modify? I would never know where to look for this value.

In fact, can we pull this value from our design tokens?

Reply:
This is our height of an autocomplete option. I don't believe there's a token for this anywhere.
Happy to move it somewhere more accessible - thoughts where it should go?

Reply:
No clue. I'd talk to @jordankoschei-okta about it and see where we can go.

Reply:
@KevinGhadyani-Okta is this a blocker right now from shipping this change? If so, please suggest a change, otherwise let's get this merged.

Reply:
Not a blocker. Something we need to consider.

Reply:
Let's revisit this? I'm happy to make a ticket for it and come back to it once we have a decision on it.